cli/azd/CONTRIBUTING.md

```diff
@@ -79,7 +79,9 @@ Run `golangci-lint run ./...`
 
 ## Spell checking
 
-Install [cspell](https://cspell.org/) and then run `cspell lint "**/*.go" --relative --config ./.vscode/cspell.yaml`
+1. Install [cspell](https://cspell.org/)
+2. CD to /cli/azd
+3. Run `cspell lint "**/*.go" --relative --config ./.vscode/cspell.yaml`
```

Comment:
Not necessarily for this PR, but we should also mention the cspell extension for VS Code, which allows you to track spelling issues "on the fly", without having to run cspell manually.

Reply:
It is part of the recommended extensions: https://github.com/Azure/azure-dev/blob/main/cli/azd/.vscode/extensions.json#L6

Reply:
I created an issue to track this.